src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./card-image.css">
+    <link rel="stylesheet" href="./card-title.css">
+    <link rel="stylesheet" href="./card-code.css">
+    <link rel="stylesheet" href="./card-rate.css">
+    <link rel="stylesheet" href="./card-price.css">
+    <link rel="stylesheet" href="./card-button.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card-image">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac"
+          class="card-image__img"
+        >
+      </div>
+      <div class="card-title">
+        <h2 class="card-title__header">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      </div>
+      <div class="card-code">
+        <p class="card-code__number">Код товара: 195434</p>
+      </div>
+      <div class="card-rate">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card-rate__amount">Отзывов: 5</div>
+      </div>
+      <div class="card-price">
+        <p class="card-price__title">Цена:</p>
+        <p class="card-price__price">69 999 грн</p>
+      </div>
+      <a
+        href="#"
+        class="card-button"
+        data-qa="hover"
+      >
+        <button class="card-button__btn">Купить</button>
```

Comment:
its better to make image, title, code, rate, etc as  elements of BEM block card.
then you will have possibility to use card block wherever you want